source/fundamentals/messaging-patterns/blackboard.md

```diff
@@ -1,8 +1,50 @@
 # Blackboard
 
-Conceptual documentation for the blackboard pattern is not yet complete.  
-For the time-being, take a look at the [API reference](https://docs.rs/iceoryx2/latest)
-for information on how to use it.
+```{figure} /images/blackboard-pattern.svg
+:alt: blackboard messaging pattern
+:align: center
+:name: fig-blackboard-communication
+
+Blackboard Communication Model
+```
+
+An unidirectional messaging pattern based on a shared-memory key-value
+repository which can be modified by (for now) one writer and read by many
+readers. Updates and read operations are made on a key basis, not on the entire
+shared memory. All keys are unique and have the same shared-memory compatible
+type; the shared-memory compatible value types can vary, but are fixed for a
+specific key.
+
+The blackboard messaging pattern is ideal for scenarios where a large number of
+readers is interested in a subset of the key-value pairs, such as global
+configuration settings adjusted at runtime or a regularly updated global state,
+with each participant only interested in a small part of it.
+
+The pattern is not suitable for cases where large payloads shall be
+communicated to multiple participants ([publish-subscribe](
+/fundamentals/messaging-patterns/publish-subscribe)) or when a bidirectional
+communication is required ([request-response](
+/fundamentals/messaging-patterns/request-response)).
+
+## Mechanism
+
+```{figure} /images/blackboard-mechanism.svg
+:alt: blackboard messaging pattern
+:align: center
+:name: fig-blackboard-mechanism
+
+Blackboard in Shared Memory
+```
+
+Writers and readers don't work directly on the key-value repository, but can
+retrieve entry handles for a specific key, enabling efficient access. These
```

Comment:
Hmm, retrieving handles is also an implementation detail. How above "can obtain writing permission for specific keys"? Does this match the theory?